### DISCUSSION TITLE
Why does the .git folder weigh 427 MiB?

### DISCUSSION THREAD

**Discussion by vibl** [refinedev/refine#2194] [General]
Why does the .git folder weigh 427 MiB? (after `git clone`)

That seems a bit much when it's stated in your doc that the bundle size is 89 KiB,

**Comment by vibl** [comment]
It can be reduced to 255 MiB with `git remote prune origin && git repack && git prune-packed && git reflog expire --expire=1.month.ago && git gc --aggressive` (from https://gitbetter.substack.com/p/how-to-clean-up-the-git-repo-and)

You probably can clean it further: https://docs.gitlab.com/ee/user/project/repository/reducing_the_repo_size_using_git.html

The number of objects is also very high: 392149

Have you committed the node_modules directory in the past?

**Reply by salihozdemir** [reply]
Thanks @vibl, 

We didn't realize it. Maybe we pushed example's node_modules in the past. We'll research and try your suggestion for reduced the data.
